lib/diamonds.ts

```diff
@@ -0,0 +1,327 @@
+import {
+  IDiamondReadable,
+  IDiamondWritable,
+} from '@solidstate/typechain-types';
+import { Contract, constants } from 'ethers';
+
+const Table = require('cli-table3');
+
+type Diamond = IDiamondReadable & IDiamondWritable;
+
+export interface Facet {
+  target: string;
+  selectors: string[];
+}
+
+enum FacetCutAction {
```

Comment:
Export this and use it in the test cases that need `FacetCutAction`.

Reply:
https://github.com/solidstate-network/solidstate-solidity/pull/158/commits/9d3b06061fdc6bc420fc00d11bec97ff7995c6b7

Reply:
See #176